2020-07-03 12:04:39.4583|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 12:04:39.7867|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 12:04:39.8131|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 12:04:39.8366|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 12:04:39.9251|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 12:06:45.8893|20102|Microsoft.EntityFrameworkCore.Database.Command|ERROR|Failed executing DbCommand (214ms) [Parameters=[@p0='?' (DbType = Int32), @p1='?' (Size = 50), @p2='?' (Size = 50), @p3='?' (DbType = Int32), @p4='?' (Size = 50), @p5='?' (Size = 50)], CommandType='Text', CommandTimeout='30']
SET NOCOUNT ON;
INSERT INTO [PARTNER] ([ID_PARTNERA], [AdrIsporuke], [AdrPartnera], [ID_TVRTKE], [mbr], [TipPartnera])
VALUES (@p0, @p1, @p2, @p3, @p4, @p5); 
2020-07-03 12:06:49.3673|10000|Microsoft.EntityFrameworkCore.Update|ERROR|An exception occurred in the database while saving changes for context type 'OZO.Models.PI09Context'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert explicit value for identity column in table 'PARTNER' when IDENTITY_INSERT is set to OFF.
   at Microsoft.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at Microsoft.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at Microsoft.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at Microsoft.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at Microsoft.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at Microsoft.Data.SqlClient.SqlDataReader.get_MetaData()
   at Microsoft.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at Microsoft.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean isAsync, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at Microsoft.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry, String method)
   at Microsoft.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at Microsoft.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at Microsoft.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
ClientConnectionId:ecae8080-b506-4313-8ac8-2dff91d1fd9c
Error Number:544,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(DbContext _, Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess) An error occurred while updating the entries. See the inner exception for details.
2020-07-03 12:11:25.0905|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 12:11:25.4820|0|OZO.Program|DEBUG|init main 
2020-07-03 12:11:25.8438|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 12:11:25.9027|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 12:11:25.9694|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 12:11:26.0208|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 12:11:26.0586|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 12:11:32.0132|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 12:30:41.8848|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 12:30:42.3455|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 12:30:42.4608|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 12:30:42.5437|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 12:30:42.6352|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 13:35:12.7365|20102|Microsoft.EntityFrameworkCore.Database.Command|ERROR|Failed executing DbCommand (135ms) [Parameters=[@p0='?' (DbType = Int32)], CommandType='Text', CommandTimeout='30']
SET NOCOUNT ON;
DELETE FROM [PARTNER]
WHERE [ID_PARTNERA] = @p0;
SELECT @@ROWCOUNT; 
2020-07-03 13:35:13.6082|10000|Microsoft.EntityFrameworkCore.Update|ERROR|An exception occurred in the database while saving changes for context type 'OZO.Models.PI09Context'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The DELETE statement conflicted with the REFERENCE constraint "FK_OSOBA". The conflict occurred in database "PI-09", table "dbo.OSOBA", column 'ID_OSOBE'.
The statement has been terminated.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:2fadf1d5-8f43-47c8-bf56-8a53548c0a6a
Error Number:547,State:0,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(DbContext _, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken) An error occurred while updating the entries. See the inner exception for details.
2020-07-03 13:37:49.3450|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 13:37:49.5121|0|OZO.Program|DEBUG|init main 
2020-07-03 13:37:49.6714|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 13:37:49.6910|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 13:37:49.7079|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 13:37:49.7326|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 13:37:49.7524|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 13:37:57.5621|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 13:40:50.4857|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 13:40:50.5896|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 13:40:50.6114|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 13:40:50.6351|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 13:40:50.6525|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 13:44:46.0663|20102|Microsoft.EntityFrameworkCore.Database.Command|ERROR|Failed executing DbCommand (30ms) [Parameters=[@p0='?' (DbType = Int32)], CommandType='Text', CommandTimeout='30']
SET NOCOUNT ON;
DELETE FROM [PARTNER]
WHERE [ID_PARTNERA] = @p0;
SELECT @@ROWCOUNT; 
2020-07-03 13:44:46.2548|10000|Microsoft.EntityFrameworkCore.Update|ERROR|An exception occurred in the database while saving changes for context type 'OZO.Models.PI09Context'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The DELETE statement conflicted with the REFERENCE constraint "FK_OSOBA". The conflict occurred in database "PI-09", table "dbo.OSOBA", column 'ID_OSOBE'.
The statement has been terminated.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:9241d27b-2cb5-4b13-a3b9-50c6d3a24401
Error Number:547,State:0,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(DbContext _, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken) An error occurred while updating the entries. See the inner exception for details.
2020-07-03 13:46:55.1274|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 13:46:55.2671|0|OZO.Program|DEBUG|init main 
2020-07-03 13:46:55.5113|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 13:46:55.5438|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 13:46:55.5759|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 13:46:55.6080|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 13:46:55.6394|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 13:47:02.3271|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 13:53:58.8928|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 13:53:59.0387|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 13:53:59.0578|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 13:53:59.0751|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 13:53:59.0921|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 13:55:36.6123|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 13:55:36.6975|0|OZO.Program|DEBUG|init main 
2020-07-03 13:55:36.8072|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 13:55:36.8243|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 13:55:36.8414|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 13:55:36.8581|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 13:55:36.8747|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 13:58:45.3307|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 14:00:21.1765|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 14:00:21.3406|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 14:00:21.3648|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 14:00:21.3872|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 14:00:21.4160|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 14:17:33.4902|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 14:17:34.2453|0|OZO.Program|DEBUG|init main 
2020-07-03 14:17:34.6366|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 14:17:34.6530|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 14:17:34.7277|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 14:17:34.7669|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 14:17:34.9138|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 14:24:50.8011|10100|Microsoft.EntityFrameworkCore.Query|ERROR|An exception occurred while iterating over the results of a query for context type 'OZO.Models.PI09Context'.
System.Data.SqlTypes.SqlNullValueException: Data is Null. This method or property cannot be called on Null values.
   at Microsoft.Data.SqlClient.SqlBuffer.ThrowIfNull()
   at Microsoft.Data.SqlClient.SqlBuffer.get_String()
   at Microsoft.Data.SqlClient.SqlDataReader.GetString(Int32 i)
   at lambda_method(Closure , QueryContext , DbDataReader , ResultContext , Int32[] , ResultCoordinator )
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext() Data is Null. This method or property cannot be called on Null values.
2020-07-03 14:24:51.2526|1|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|ERROR|An unhandled exception has occurred while executing the request. Data is Null. This method or property cannot be called on Null values.
2020-07-03 14:25:32.0963|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 14:28:12.9583|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 14:28:13.7242|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 14:28:13.7576|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 14:28:13.7742|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 14:28:13.7952|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 14:28:13.8151|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 14:28:13.9184|0|OZO.Program|DEBUG|init main 
2020-07-03 14:28:14.0029|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 14:28:14.0178|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 14:28:14.0322|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 14:28:14.0463|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 14:28:14.0839|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 14:28:17.4236|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 14:32:36.2670|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 14:32:36.3870|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 14:32:36.4047|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 14:32:36.4192|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 14:32:36.4336|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 14:53:42.6995|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
2020-07-03 14:53:43.2479|0|OZO.Program|DEBUG|init main 
2020-07-03 14:53:43.4529|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: https://localhost:5001 
2020-07-03 14:53:43.4529|0|Microsoft.Hosting.Lifetime|INFO|Now listening on: http://localhost:5000 
2020-07-03 14:53:43.4917|0|Microsoft.Hosting.Lifetime|INFO|Application started. Press Ctrl+C to shut down. 
2020-07-03 14:53:43.5195|0|Microsoft.Hosting.Lifetime|INFO|Hosting environment: Development 
2020-07-03 14:53:43.5346|0|Microsoft.Hosting.Lifetime|INFO|Content root path: C:\Users\Iva\Desktop\FSRE_sem6\PI\vjezba\OZO 
2020-07-03 14:57:54.8426|0|Microsoft.Hosting.Lifetime|INFO|Application is shutting down... 
